src/Common/tests/System/Collections/IEnumerable.Generic.Tests.cs

```diff
@@ -450,23 +441,6 @@ public void IEnumerable_Generic_Enumerator_MoveNextFalseAfterEndOfCollection()
         #region Enumerator.Current
 
         [Fact]
-        public void IEnumerable_Generic_Enumerator_CurrentThrowsAfterEndOfCollection()
-        {
-            if (Enumerator_Current_UndefinedOperation_Throws)
-            {
-                RepeatTest(
-                    (enumerator, items) =>
-                    {
-                        while (enumerator.MoveNext())
-                        {
-                        }
-                        Assert.Throws<InvalidOperationException>(
-                            () => enumerator.Current);
-                    });
-            }
-        }
-
-        [Fact]
         public void IEnumerable_Generic_Enumerator_Current()
```

Comment:
This test not moved down to List testing. I removed it because we already have such test in `VerifyEnumerator` method. I removed double testing of the same.